plugins/main/bridge/bridge.go

```diff
@@ -374,11 +375,47 @@ func setupVeth(netns ns.NetNS, br *netlink.Bridge, ifName string, mtu int, hairp
 		if err != nil {
 			return nil, nil, fmt.Errorf("failed to setup vlan tag on interface %q: %v", hostIface.Name, err)
 		}
+
+		if disableDefaultVlan {
+			err = removeDefaultVlan(hostVeth, vlanID)
+			if err != nil {
+				return nil, nil, fmt.Errorf("failed to remove default vlan on interface %q: %v", hostIface.Name, err)
+			}
+		}
 	}
 
 	return hostIface, contIface, nil
 }
 
+func removeDefaultVlan(hostVeth netlink.Link, vlanID int) error {
+	vlanInfo, err := netlink.BridgeVlanList()
+	if err != nil {
+		return err
+	}
+
+	var defaultVlan uint16
+out:
+	for deviceID, brVlanInfo := range vlanInfo {
+		if deviceID == int32(hostVeth.Attrs().Index) {
```

Comment:
why do we iterate the map when we can just get the corresponding slice from it ? 
Something like: 
```golang
brVlanInfo, isDeviceIDPresent := vlanInfo[int32(hostVeth.Attrs().Index)]
if !isDeviceIDPresent {
    return nil
}
...
```

Am I missing something ?

Reply:
Of course, thanks. I'll change it.

Reply:
done